hil/cli/misc.py

```diff
@@ -0,0 +1,109 @@
+"""Miscellaneous commands go here"""
+import click
+from hil.cli.client_setup import setup_http_client
+
+C = setup_http_client()
+
+
+@click.command(name='serve', help='Run a development HIL server')
```

Comment:
While we're at it, why not rename this to `dev-server`? I think it might be a bit more clear.